steps/hockeyapp-deploy-android/1.2.0/step.yml

```diff
@@ -0,0 +1,216 @@
+title: HockeyApp Android Deploy
+summary: Deploy your awesome Android app to HockeyApp.
+description: |-
+  Deploy an Android application to [HockeyApp](http://hockeyapp.net),
+  add notes even in markdown format, notify testers, restrict download.
+
+  Register a HockeyApp account at [http://hockeyapp.net/](http://hockeyapp.net)
+  and create an app to utilize this step.
+  You also need to create a *HockeyApp Token* for your app.
+
+  If app_id is provided the apk will be upload to the specified app on HockeyApp,
+  otherwise HockeyApp will decide based on the package ID, if it's a new app, or update.
+website: https://github.com/bitrise-io/steps-hockeyapp-android-deploy
+source_code_url: https://github.com/bitrise-io/steps-hockeyapp-android-deploy
+support_url: https://github.com/bitrise-io/steps-hockeyapp-android-deploy/issues
+published_at: 2017-12-04T14:58:00.22548087+01:00
+source:
+  git: https://github.com/bitrise-steplib/steps-hockeyapp-android-deploy.git
+  commit: 5e820f529adf02e6b5ba63d982571f7c9ce89974
+host_os_tags:
+- ubuntu
+- osx-10.9
+- osx-10.10
+project_type_tags:
+- android
+- xamarin
+- cordova
+- ionic
+- react-native
+type_tags:
+- deploy
+toolkit:
+  go:
+    package_name: github.com/bitrise-steplib/steps-hockeyapp-android-deploy
+deps:
+  brew:
+  - name: go
+  apt_get:
+  - name: golang
+    bin_name: go
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+run_if: .IsCI
+inputs:
+- apk_path: $BITRISE_APK_PATH
+  opts:
+    description: 'The paths are separated with `|` character, eg: `app-armeabi-v7a-debug.apk|app-mips-debug.apk|app-x86-debug.apk`'
+    is_required: true
+    summary: ""
+    title: apk file path
```

Comment:
Shouldn't it include info that there can be multiple paths? e.g. `APK file path or paths`